mcc/odin/odin/snapshot_list.go

```diff
@@ -18,6 +20,14 @@ func ListSnapshots(
 	if err != nil {
 		return
 	}
+	sort.Slice(
```

Comment:
maybe a comment in the code would be useful? sth like `# order by snapshot creation time, descending`...